### ISSUE TITLE
Releases and changelog

### ISSUE THREAD

**Issue by romlly** [eurostat/searoute#37]
Is there a place where I can find every releases and see a changelog ?

In GH there is a v1.0 release associated with a tag and in master I see a v2.1, I'm confused about which version should be used.

Thank you

**Comment by mlthlschr** [comment]
To me, it looks like there are some issues in the usage of git in this repo. Instead of flagging commits as release, binaries were pushed in a subfolder of the project. Of course, it is possible to do that, but it also unnecessarily increases the size of the repo. I just cloned it, and I needed to download > 300 MB, from which the majority is located within the `.git` folder. 

Maybe a cleanup should be done, which of course will result in history changes. But at least there would be clear releases and less garbage on the harddrive.

See also this on [stackoverflow](https://stackoverflow.com/questions/11050265/remove-large-pack-file-created-by-git/11277929).

Okay, this will not answer your question. But I suppose the version found in the releases folder (2.1 from Apr 27) is the one which is more current than the only flagged release (2019).
